src/database/admin.rs

```diff
@@ -360,10 +361,21 @@ impl AdminDatabase {
             ":pubkey": node.pubkey,
             ":status": node.status
         })?;
-
+        dbg!("Wrote node to the database");
```

Comment:
I'll remove some of these `dbg!`s.  I went a little crazy with them trying to understand why the code was failing.